src/mpi_find_approximation_set.cpp

```diff
@@ -183,6 +189,10 @@ int main(int argc, char** argv) {
     std::vector<unsigned int> rowToRank = MpiOrchestrator::getRowToRank(appData, seed);
 
     Timers timers;
```

Comment:
Delete this if it's not in use.

Reply:
Ah, double checked why this is still here. We should do something so that we don't have to record every timing before entering into `randgreedi` or `streaming` twice. Even splitting the timing class into a `beforeExperimentTimers` class and a `duringExperimentTimers` class would be fine I think.